### DISCUSSION TITLE
Convert Model to Edge TPU TFlite Format for Google Coral | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1716] [Q&A]
# Convert Model to Edge TPU TFlite Format for Google Coral | Seeed Studio Wiki

This wiki demonstrates how to compile a tensorflow model or pytorch model to edge tup model, and run it.

https://wiki.seeedstudio.com/install_m2_coral_to_rpi5/

**Comment by wb666greene** [comment]
You'll get much better results with the yolov8s model and the Coral TPU if you resize the model from 640x640 to 512x512 as then the bulk of the processing will happen on the TPU, at 640x640 too much is done on the CPU which really increases the inference time.

            model = YOLO('yolo8/yolov8s')
            model.export(format="edgetpu", imgsz=512)

More details are on the Ultralytics site, but this should  greatly improve your inference time, I get about 10 fps using a lame Celeron system with an M.2 TPU and yolov8s imgsz=512
